ubuntu/debian/control

```diff
@@ -43,7 +45,9 @@ Architecture: any
 Section: libdevel
 Depends: libgz-cmake4-dev,
          libgz-utils3-dev,
+         libgz-utils3-log-dev
```

Comment:
and a comma here?

Reply:
fixed. [5ccdf78](https://github.com/gazebo-release/gz-common6-release/pull/5/commits/5ccdf78222cb9a8756cee8ac99dbfb3a4702aa4f)